### PR TITLE
Add xditya/GrammyCloneableBot

### PR DESCRIPTION
**Pull request by xditya**

### CODE REVIEW

**Comment by xditya** [comment]
@KnorpelSenf

File: README.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [waptik/grammy-on-deta-template](https://github.com/waptik/grammy-on-deta-template) - A template to start a grammY bot and deploy it on [Deta](https://www.deta.sh). [ᴜsᴇ](https://t.me/GrammyOnDetaBot)
 - [ShinkarenkoMaxim/webapp-vanilla](https://github.com/ShinkarenkoMaxim/webapp-vanilla) - Telegram Web App starter template.
 - [Borodutch/telegram-bot-starter](https://github.com/Borodutch/telegram-bot-starter) - Telegram bot starter based on grammY.
-
+- [xditya/GrammyCloneableBot](https://github.com/xditya/GrammyCloneableBot) - A template for a clone-able telegram bot, i.e., one which allows users to run their own instance of the bot within your bot.
```
Comment (roj1512) [inline README.md:56]: Do you have any other idea for the description that is as clear as the rest, or you like to see mine?

Comment (xditya) [reply README.md:56]: You can suggest, would be great!

Comment (roj1512) [reply README.md:56]:
```suggestion
- [xditya/GrammyCloneableBot](https://github.com/xditya/GrammyCloneableBot) - A template for bots that can be cloned.
```

**Review by roj1512 — APPROVED** [review]
LGTM.

**Pull request merged**